health/health.go

```diff
@@ -705,7 +706,20 @@ func (t *Tracker) SetIPNState(state string, wantRunning bool) {
 	t.mu.Lock()
 	defer t.mu.Unlock()
 	t.ipnState = state
+	prevWantRunning := t.ipnWantRunning
 	t.ipnWantRunning = wantRunning
+
+	if wantRunning && !prevWantRunning && t.ipnWantRunningLastTrue.IsZero() {
+		// The first time we see wantRunning=true and it used to be false, it means the user requested
+		// the backend to start. We store this timestamp and use it to silence some warnings that are
+		// expected during startup.
+		t.ipnWantRunningLastTrue = time.Now()
+		t.setUnhealthyLocked(warmingUpWarnable, nil)
+	} else if !wantRunning {
+		// Reset the timer when the user decides to stop the backend.
+		t.ipnWantRunningLastTrue = time.Time{}
+	}
```

Comment:
I'd expect some time.Timer/AfterFunc in here somewhere to toggle the "warming up" timer (or evaluate the health state) in 5 second so the GUI switches from the "Tailscale is starting" to whatever the real error is in 5 seconds. Where's that?

Reply:
selfCheckLocked() is called so often that it felt redundant, but I can throw that in

Reply:
added